project/Build.scala

```diff
@@ -299,6 +299,10 @@ object Build {
     },
     */
 
+    scalacOptions ~= { old =>
+      if (old.contains("-Ysemanticdb")) old else old :+ "-Ysemanticdb"
```

Comment:
I'm against this if it slows down the build. Instead, this option can be added to the `tastyBootstrap` test which compiles the whole compiler too: https://github.com/lampepfl/dotty/blob/363dd03087819d5cfd08b6741cbc75618c17dad9/compiler/test/dotty/tools/dotc/CompilationTests.scala#L230 (I'm surprised this isn't actually compiled with Ycheck:all too!)

Reply:
Thanks, I've removed this setting and changed the settings of tastyBootstrapped to add -Ysemanticdb